substanced/folder/generate/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
You mention that it should not be mixed up with files that we do mean to serve.

I thought that I had addressed this originally by putting all the support files into a separate 'generate' subdirectory:
- it's not served up statically, because it is separate from static
- it cannot be imported from python, because there is no __init__.py (or any python file) in it.

Nevertheless, since the location is wrong in your opinion, what would be the correct subdirectory to put them to?